packages/nouns-webapp/src/wrappers/subgraph.ts

```diff
@@ -19,6 +19,28 @@ export interface IBid {
   };
 }
 
+interface ProposalVote {
+  supportDetailed: 0 | 1 | 2;
+  voter: {
+    id: string;
+  };
+}
+
+export interface ProposalVotes {
+  votes: ProposalVote[];
+}
+
+export interface DelegateNouns {
```

Comment:
Non-blocking nit:

Assuming I'm understanding this correctly this represents 1 address (`id`) to whom a series of nouns has been delegates (`nounsRepresented`). If that's true, I wonder if the name could be changed to make it clear this is the delegated address vs being a Noun or something.